### DISCUSSION TITLE
Drupal 11

### DISCUSSION THREAD

**Discussion by owen-soak** [GetDKAN/dkan#4294] [Q&A]
Hello, 

New to dkan and couldn't find out if dkan supports the latest drupal release (11) yet? Has anyone tried using it and been succesful?

**Comment by paul-m** [comment]
I just spent a little time trying to make it work with D11, and unfortunately it's going to take a little work to get all our dependencies happening.

So no, you won't be successful as things stand, though of course any PRs related to this are welcome. :-)

We do have Drupal 10.3.x in our testing matrix, which gets you pretty close to 11. https://app.circleci.com/pipelines/github/GetDKAN/dkan/5834/workflows/812b975e-4e20-416b-8758-f85b8c00138b/jobs/39768

**Reply by owen-soak** [reply]
Thank you for the response!

**Comment by dafeder** [comment]
Hi @owen-soak FYI I have an issue and PR going a little more actively for this now, see #4356 and #4354
